.prettierrc

```diff
@@ -6,5 +6,5 @@
   "printWidth": 80,
   "tabWidth": 2,
   "useTabs": false,
-  "proseWrap": "always"
+  "proseWrap": "preserve"
```

Comment:
not related to this PR, please let's remove it from here and have a separate PR/discussion on this.

Reply:
Without this the changes to dvc.yaml samples in this PR fail checks though, so I consider it related. Is it OK if I have to skip git hooks and then merge this PR with failing checks?

Reply:
What are the changes that fail?

Reply:
I've reverted this. The changes that fail are [here](https://github.com/iterative/dvc.org/pull/2052/files#diff-584bff5a435baee84112ffda33bc2d9dc94e1512e7306322f43655d5e7593876R283-R287) (see new restyled version: https://github.com/iterative/dvc.org/pull/2061/files).

Reply:
p.s. we can discuss there: https://github.com/iterative/dvc.org/pull/2061#pullrequestreview-558975989. Resolving here 🙂